modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -799,6 +953,57 @@
 ---
 -- ClCompile group
 ---
+
+	m.fileConfigFunction = function(fcfg, condition)
+
+		return {
+			m.excludedFromBuild,
+			m.objectFileName,
+			m.clCompilePreprocessorDefinitions,
+			m.clCompileUndefinePreprocessorDefinitions,
+			m.optimization,
+			m.forceIncludes,
+			m.precompiledHeader,
+			m.enableEnhancedInstructionSet,
+			m.additionalCompileOptions,
+			m.disableSpecificWarnings,
+			m.treatSpecificWarningsAsErrors,
+			m.basicRuntimeChecks,
+			m.exceptionHandling,
+			m.compileAsManaged,
+			m.compileAs,
+			m.runtimeTypeInfo,
+			m.warningLevelFile,
+			m.compileAsWinRT,
+			m.externalWarningLevelFile,
+			m.externalAngleBrackets,
+		}
+
+	end
+
+	m.linuxFileConfigFunction = function(fcfg, condition)
+
+		return {
+			-- Common
+			m.excludedFromBuild,
+			m.objectFileName,
+			m.clCompilePreprocessorDefinitions,
+			m.clCompileUndefinePreprocessorDefinitions,
+			m.optimization,
+			m.forceIncludes,
+			--m.enableEnhancedInstructionSet,
```

Comment:
Why commented?

Reply:
I need to review this one as well. I'm not sure how these should be handled, I'll review it too

Reply:
I have left this one out for now, as I'm not sure how to best tackle it. The builtin xml tags that the MSVC compiler uses don't work, but are also very poor in terms of expressing multiple versions of SSE. If we wanted this to be correct we'd have to have a function that properly passes in -msse4.1 and all the rest. I'll leave it for a future PR